introduction.md

```diff
@@ -1,2 +1,9 @@
 I am Mark Master and I am learning to master GitHub!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?